cilium/metadata.csv

```diff
@@ -1,149 +1,212 @@
+cilium.operator.ipam.api.duration.seconds.count,count,,request,,"[OpenMetricsV1 and V2] Duration of interactions with external IPAM API",0,cilium,ipam api duration count,
+cilium.operator.ipam.api.duration.seconds.bucket,count,,second,,"[OpenMetricsV2] Duration of interactions with external IPAM API",0,cilium,ipam api duration bucket,
+cilium.operator.ipam.api.rate_limit.duration.seconds.sum,count,,second,,"[OpenMetricsV1 and V2] Sum of duration of rate limiting while accessing external IPAM API",0,cilium,ipam api rate limit duration sum,
+cilium.operator.ipam.api.rate_limit.duration.seconds.count,count,,request,,"[OpenMetricsV1 and V2] Count of duration of rate limiting while accessing external IPAM API",0,cilium,ipam api rate limit duration count,
+cilium.operator.ipam.api.rate_limit.duration.seconds.bucket,count,,second,,"[OpenMetrics V2] Sample of duration of rate limiting while accessing external IPAM API",0,cilium,ipam api rate limit duration bucket,
+cilium.operator.ipam.allocation_ops,count,,operation,,"[OpenMetrics V1] Count of IP allocation operations",0,cilium,ipam allocation ops,
+cilium.operator.ipam.allocation_ops.count,count,,operation,,"[OpenMetrics V2] Count of IP allocation operations",0,cilium,ipam allocation ops,
+cilium.operator.ipam.release_ops,count,,operation,,"[OpenMetrics V1] Count of IP release operations",0,cilium,ipam release ops,
+cilium.operator.ipam.release_ops.count,count,,operation,,"[OpenMetrics V1] Count of IP release operations",0,cilium,ipam release ops,
+cilium.operator.ipam.interface_creation_ops,count,,operation,,"[OpenMetrics V1] Count of interfaces allocated",0,cilium,ipam creation ops,
+cilium.operator.ipam.interface_creation_ops.count,count,,operation,,"[OpenMetrics V2] Count of interfaces allocated",0,cilium,ipam creation ops,
+cilium.operator.ipam.available,gauge,,unit,,"[OpenMetrics V1 and V2] Number of interfaces with addresses available",0,cilium,available ips,
+cilium.operator.ipam.available.ips_per_subnet,gauge,,unit,,"[OpenMetrics V1 and V2] Number of available IPs per subnet ID",0,cilium,available ips/subnet,
+cilium.operator.ipam.deficit_resolver.duration.seconds.sum,count,,second,,"[OpenMetrics V1 and V2] Sum of duration of deficit resolver trigger runs",0,cilium,deficit resolver duration sum,
+cilium.operator.ipam.deficit_resolver.duration.seconds.count,count,,request,,"[OpenMetrics V1 and V2] Count of duration of deficit resolver trigger runs",0,cilium,deficit resolver duration count,
+cilium.operator.ipam.deficit_resolver.duration.seconds.bucket,count,,second,,"[OpenMetrics V2] Sample of duration of deficit resolver trigger runs",0,cilium,deficit resolver duration sample,
+cilium.operator.ipam.deficit_resolver.queued.count,count,,unit,,"[OpenMetrics V1] Number of queued triggers",0,cilium,deficit resolver queued count,
+cilium.operator.ipam.deficit_resolver.queued.total,count,,unit,,"[OpenMetrics V2] Number of queued triggers",0,cilium,deficit resolver queued count,
```

Comment:
```suggestion
cilium.operator.ipam.deficit_resolver.queued.total,count,,unit,,"[OpenMetrics V1] Number of queued triggers",0,cilium,deficit resolver queued count,
```

Not completely sure if this is the case, but it looked like other `.total` metrics were V1 only?